jwst/outlier_detection/tests/test_outlier_detection.py

```diff
@@ -195,42 +195,55 @@ def test_outlier_step_no_outliers(we_three_sci, tmp_cwd):
     """Test whole step, no outliers"""
     container = ModelContainer(list(we_three_sci))
     pristine = ModelContainer([m.copy() for m in container])
-    OutlierDetectionStep.call(container)
+    OutlierDetectionStep.call(container, in_memory=True)
 
     # Make sure nothing changed in SCI and DQ arrays
     for image, uncorrected in zip(pristine, container):
         np.testing.assert_allclose(image.data, uncorrected.data)
         np.testing.assert_allclose(image.dq, uncorrected.dq)
 
 
-def test_outlier_step(we_three_sci, tmp_cwd):
+def test_outlier_step_base(we_three_sci, tmp_cwd):
     """Test whole step with an outlier including saving intermediate and results files"""
-    container = ModelContainer(list(we_three_sci))
+    container = ModelLibrary(list(we_three_sci))
 
     # Drop a CR on the science array
-    container[0].data[12, 12] += 1
+    with container:
+        zeroth = container.borrow(0)
+        zeroth.data[12, 12] += 1
+        container.shelve(zeroth)
 
     # Verify that intermediary files are removed
-    OutlierDetectionStep.call(container)
+    OutlierDetectionStep.call(container, in_memory=True)
```

Comment:
```suggestion
    OutlierDetectionStep.call(container)
```
I think this needs to be `in_memory=False` to check for the intermediate files (which aren't written if `in_memory=True`).